### PR TITLE
Fix several warnings during tests

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: ctapipe/tools/tests/test_merge.py [hunk 1]
```diff
@@ -113,7 +113,8 @@ def test_allowed_tels(tmp_path, dl1_file, dl1_proton_file):
             str(dl1_file),
             str(dl1_proton_file),
             f"--output={output}",
-            "--allowed-tels=[1,2]",
+            "--allowed-tels=1",
+            "--allowed-tels=2",
```
Comment (HealthyPear) [inline ctapipe/tools/tests/test_merge.py:117]: maybe I lost some previous discussion: why is this needed?

Comment (maxnoe) [reply ctapipe/tools/tests/test_merge.py:117]: It generated traitlets warnings. Passing a list literal is deprecated and will be removed soon

**Review by HealthyPear — APPROVED** [review]

**Review by kosack — APPROVED** [review]

**Review by HealthyPear — APPROVED** [review]

**Review by LukasNickel — APPROVED** [review]
